assets/js/blocks/checkout/inner-blocks/checkout-pickup-options-block/style.scss

```diff
@@ -50,7 +50,7 @@
 	.wc-block-components-radio-control__description,
 	.wc-block-components-radio-control__secondary-description {
 		width: 100%;
-		text-align: left;
+		text-align: left !important; // To force align left on the description for the local pickup options.
```

Comment:
On truck, the local pick-up description was aligned right.  Added `!important` to force align left.

| Before | After |
|---|---|
|![image](https://github.com/woocommerce/woocommerce-blocks/assets/11503784/817776a7-366a-4b9b-b036-b3666d583f87)|![image](https://github.com/woocommerce/woocommerce-blocks/assets/11503784/0d8d061b-967d-48ae-ba09-215bdd73366e)|

Reply:
Reverted the change as it got fixed in trunk